src/Approximations/overapproximate.jl

```diff
@@ -714,6 +714,22 @@ end
 # Overapproximation of Sparse Polynomial Zonotopes
 # ================================================
 
+#=
+Given center, (dependent) generator matrix and exponent matrix of a (simple) sparse polynomialzonotope,
+compute thew new center and generator matrix of its zonotope overapproximation.
+=#
+function _zonotope_overapprox(c, G, E)
+    cnew = copy(c)
+    Gnew = copy(G)
```

Comment:
I suggest to use an in-place method and pass copies to it, like this:
```suggestion
function _zonotope_overapprox!(c, G, E)
```

Reply:
do you mean to have both in-place and non-inplace helpers or just in-place helper and call `copy` in the exported `overapproximate` ?

Reply:
Both are fine, but the latter is sufficient. It is only a suggestion, though.

Reply:
looking at how the `overapproximate` is written, maybe having both `_zonotope_overapprox!` and `_zonotope_overapprox` might be clearer?